cmd/jx/main.go

```diff
@@ -6,8 +6,7 @@ import (
 
 	"foxygo.at/jsonnext"
 	jxkong "foxygo.at/jsonnext/kong"
-	"github.com/alecthomas/kong"
-	"github.com/google/go-jsonnet"
+	jsonnet "github.com/google/go-jsonnet"
```

Comment:
I wonder if you can rework this to not depend on `jxkong "foxygo.at/jsonnext/kong"` as well (at least for the flag case).

But maybe that's ott.

Reply:
that was my original design where I had `jsonnext.Config` in there and in parseCLI, I created a temp struct with `jxkong.Config` embedded alongside `jsonnext.Config` - the idea that the former was a hidden implementation detail. But i changed that because you said it was too complicated.

But, for the flag case, i did check the compiled binaries, and it appears it does not link in kong and its dependencies, so i'm not too concerned.

Reply:
I see... you shouldn't listen to me. I am saying a lot of stuff....